.github/workflows/test.yml

```diff
@@ -94,22 +107,54 @@ jobs:
         - zsh
         - dash
         - ksh
+        - mksh
+        - yash
+        - 'busybox sh'
+        - gash
+        - mrsh
+        #- 'toybox toysh'
+        #- oil
```

Comment:
What's up with these? Did they not work and we should delete them?

Reply:
These are completely defunct shells that have such serious issues with basic POSIX syntax (e.x. toybox doesn't support `||` or multiline strings and oil has issues with variables requiring a special oil-only syntax in order to be assigned) that supporting these niche shells would be practically impossible.

NONETHELESS, as these projects mature over time, it is possible that they may eventually gain the needed posix syntax in order to be worthy of consideration, so I'm leaving our options open for the future.